views/components/blocks.njk

```diff
@@ -1,20 +1,27 @@
 {% macro projectBlock(project, accent) %}
-    <div class="card{% if accent %} {{ accent }} a--border-top{% endif %}">
-        <div class="card__media">
+    <div class="project-card {% if accent %} {{ accent }} a--border-top{% endif %}">
+        <div class="project-card__media">
             {% if project.link %}<a href="{{ project.link }}">{% endif %}
-                <img src="{{ project.image | getCachebustedPath }}" class="grid__item__poster" alt="{{ project.description }}" />
+                <img
+                    class="project-card__image"
+                    src="{{ project.image | getCachebustedPath }}"
+                    alt="{{ project.description }}"
+                />
             {% if project.link %}</a>{% endif %}
         </div>
-        <div class="card__body">
-            <h4 class="t4">{{ project.name }}</h4>
+        <div class="project-card__body">
+            <h4 class="project-card__header t4">{{ project.name }}</h4>
+
             {% if project.amount %}
-                <p class="card__subheader t7 accent--blue a--text">Grant amount {{ project.amount }}</p>
+            <p class="project-card__subheader t7 accent--blue a--text">
+                Grant amount {{ project.amount }}
```

Comment:
Could you change this to "Grant: {{ project.amount }}"? Then it won't need translation!